api/src/routes/users.js

```diff
@@ -2,17 +2,21 @@
  * This module defines endpoints for the "/users" route and exports its corresponding Router
  */
 const usersRouter = require('express').Router();
-
 const usersController = require('../controllers/users');
+const authFunctions = require('../middlewares/authFunctions');
 
 usersRouter.get('/', usersController.getUsers);
 
+usersRouter.get('/current', authFunctions.ensureAuthenticated, usersController.getCurrentUser);
+
```

Comment:
Related to the previous comment, this won't be needed if we decide to remove this functionality.

Reply:
Same as above, I think this is a good example for people to see. As no one else would know how to use authFunctions yet.